packages/DataGrid/src/DataGrid.styles.js

```diff
@@ -128,7 +128,7 @@ export const Cell = styled.div.attrs(({ hasZebraStripes, rowIndex, borderType })
   display: flex;
   font-size: ${tokens.fontSize.default};
   justify-content: flex-start;
-  line-height: 1;
+  line-height: 18px;
```

Comment:
mmm this will not work if `${tokens.fontSize.default}` change their default, does `1.1` works?